base/server/python/pki/server/instance.py

```diff
@@ -695,15 +695,19 @@ def cert_import(
 
         # If cert_file is not provided, load the cert from
         # /var/lib/pki/<instance>/conf/certs/<cert_id>.crt
-        if not cert_file:
+        if not cert_file and cert_id:
```

Comment:
If both are not specified `cert_file` is None and there should be a type error on line 703. Is a check useful?

Reply:
Good catch. I've updated the code to raise an exception if neither them are specified.